envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/assertions/ResponseAssertions.kt

```diff
@@ -31,3 +31,8 @@ fun ObjectAssert<Response>.isFrom(echoServiceExtension: EchoServiceExtension): O
     }
     return this
 }
+
+fun ObjectAssert<Response>.hasXEnvoyUpstreamRemoteAddressFrom(echoServiceExtension: EchoServiceExtension): ObjectAssert<Response> {
```

Comment:
`hasXEnvoyUpstreamRemoteAddressFrom` in `EnvoyControlTestConfiguration` is not used anywhere else. You can remove it.

Reply:
If it's the only usage I'd put it into the test test file instead of global `ResponseAssertions`.

Reply:
@chemicL WDYF?

Reply:
it is moved now!